### ISSUE TITLE
Collected Frontend issues to fix before master push

### ISSUE THREAD

**Issue by heliogabal** [saurabhd/hk_realestate#47]
Hi Darshi, 
I tested the frontend part while not being logged in and found the following issues still needing a fix before we can call the site stabilized: 
## General / Desktop:

~~get rid of zoom indicator on map~~
~~on zoom 12, have less clustered markers (like on 13, or even 14)~~

~~Old Building / New Building not translated~~
~~if I select a borough or category on buy page, search bar spans two lines~~
- once I used the filter on buy page, I get the result counter saying X offers and reset. That should translate to German: X Angebote and Zurücksetzen. ~~update Oct 21.: this got worse, not I see an Array printed in screen when I activate the search box. And the Buy isn't translated any longer. It's really tiring having to check, refresh strings and re-translate everytime...~~

~~on favorites, the button says Inquiry instead of Anfrage, and the hover-color is black with underline. Should match other buttons. The string  `<a class="inquiry-button btn ng-lightbox" href="/anfrage?field_angefragte_wohnung=&destination=/favorites&width=600&height=600" >Inquiry</a>`  is not translated, and I can't translate it because of unallowed HTML. Apart from that, there are no untranslated Inquiry strings available, and I don't think the one mentioned before is the actual string we are looking for.~~

~~The favorites inquiry doesn't work any longer. AJAX error with popup.~~

~~if I toggle the search bar on for example /mieten I get English search labels that are smaller than on /buy~~
~~the search toggle should also work on object and apartment pages. -> works, but is overlapped by address, so not usable right now.~~ 

~~the toggled search bar should have more withespace underneath to stand out some more (slightly more than from the headline to the bar).~~

~~On the object page, building type shouldn't be visible.~~
## Responsive:
- Some markers are not clickable on map, popup doesn't open (for example the 12 apartments on Spenerstr. in Moabit or K55 with 21 in Mitte)

~~the keyboard still opens on selecting a search select, whch makes it next to impossible to select something~~
~~scale and attribution is still underneath the footer instead of on top of it.~~
~~floorplan is still too big so it goes on next line~~

**Comment by darshi-shah** [comment]
Hi Rainer,

CSS changes are pushed on develop branch. Please take pull. We are working on point 2,5.

Thanks

**Comment by heliogabal** [comment]
Hi Darshi, 
ok, I'll pull.

**Comment by heliogabal** [comment]
Pulled and tested on desktop. On the favorites page, the image sprites position got messed up. There is a close or caret icon instead of the star.

**Comment by darshi-shah** [comment]
Hi Rainer,

This is fixed along with translation issue. Please take pull and refresh strings.

Also, for clustering, can we make zoom 13 as initial level ? 

We are disabling keyboard for all select lists on search filters. Please confirm this.

Thanks
Darshi

**Comment by heliogabal** [comment]
Hi Darshi, 
I pulled and refreshed strings. 

No, we need to stay on zoom 12 to see the big picture and get all markers on initial page. Leave it be for the time being, we can have another look at that after feedback from the client. 

Yes, as I see it, we don't need keyboard on the select search filters, at least for tablets and mobile. On the desktop, the select2 library provides additional value by being able to also type in the wanted caegory, but if that's not easily separatable, it is better to deactivate that as well on the desktop, rather than having the search being unable to be used on mobile and tablet.

**Comment by darshi-shah** [comment]
Hi Rainer,

Keyboard is turned off for mobile and tablets. Please take pull on dev site.

Thnaks
Darshi

**Comment by heliogabal** [comment]
Regarding clustering, what is the problem with just setting the Marker cluster radius from 200 to say 100? That seems to do the trick, or am I missing something here?

**Comment by darshi-shah** [comment]
Hi Rainer,

Zoom level issue is fixed. Thanks for the tip!

On buy page, array issue is solved, However, reset text is not being translated. I will be checking this on monday.

Building type is hidden from object page.

As for markup click issue when we have more apartments, may be we should first fix how to show more apartments. This might be solved via that. 

Thanks

**Comment by heliogabal** [comment]
Hi Darshi, 
I'll pull and checked. All fine.

I didn't think of not being able to open the marker had to do with the number of apartments, but it seems true. There might be some left over js code in hk_theme script.js or in the hk_leaflet modules script responsible for that. 

We should see to pushing the changes to stage branch to finish the stabilization next week.

**Comment by darshi-shah** [comment]
Hi Rainer,

Translation is done and pushed on develop branch.

Thanks

**Comment by heliogabal** [comment]
Closing this and opening new issue with the remaining task.
